### ISSUE TITLE
vitest/require-test-timeout false positive for imported const timeout values

### ISSUE THREAD

**Issue by lerothias** [vitest-dev/eslint-plugin-vitest#892]
### Environment

- `eslint-plugin-vitest`: `1.6.12`
- `eslint`: `10.1.0`
- `vitest`: `4.0.18`

### Reproduction

```ts
// test-constants.ts
export const DEFAULT_VISUAL_TEST_TIMEOUT = 10_000
```

```ts
// example.visual.test.ts
import { DEFAULT_VISUAL_TEST_TIMEOUT } from './test-constants'

it(
  'renders something',
  async () => {
    expect.assertions(1)
    expect(true).toBe(true)
  },
  DEFAULT_VISUAL_TEST_TIMEOUT,
)
```

With `vitest/require-test-timeout` enabled, this reports:

> Test is missing a timeout. Add an explicit timeout.

If I replace the imported constant with a local same-file `const`, the warning goes away.

### Expected behavior

An imported `const` initialized to a numeric literal should count as an explicit timeout, same as a same-file `const`.

### Actual behavior

`vitest/require-test-timeout` appears to resolve only literals and local `const` bindings, so imported bindings are treated as if no timeout was provided.

### Notes

This looks related to the already-fixed same-file `const` support in #887 / #889, but imported constants still produce a false positive.

I checked the current rule implementation locally, and `resolveConstTimeout(...)` follows local variable definitions but does not seem to traverse `ImportSpecifier` bindings.

At least from the outside, this looks fixable rather than an inherent ESLint limitation.
